VERSION.south.cc2650

```diff
@@ -0,0 +1,2 @@
+foglamp_south_cc2650_version=1.0.0
+foglamp_version>=1.3
```

Comment:
@MarkRiddoch Can you advice here about the plugin and foglamp version? This works well with foglamp commit 2dc861e649805d002c097ae039cfb006f2ad0534. and the issue is reference here https://scaledb.atlassian.net/browse/FOGL-1626